src/libraries/System.Runtime/tests/System/TimeOnlyTests.cs

```diff
@@ -186,7 +186,7 @@ public static void SubtractOperatorTest()
             TimeOnly to2 = new TimeOnly(14, 0);
 
             Assert.Equal(new TimeSpan(3, 29, 20), to1 - to2);
-            Assert.Equal(new TimeSpan(3, 29, 20), to2 - to1);
+            Assert.Equal(new TimeSpan(20,30, 40), to2 - to1);
```

Comment:
Yeah, `14:00:00` - `10:30:40` should be `3:29:20`.

Can you please also add a test that spans over midnight, such as `01:00:00` - `23:00:00` == `02:00:00`

Reply:
ok, I will add it.